facefusion/processors/modules/lip_syncer.py

```diff
@@ -144,33 +167,53 @@ def post_process() -> None:
 
 
 def sync_lip(target_face : Face, temp_audio_frame : AudioFrame, temp_vision_frame : VisionFrame) -> VisionFrame:
-	model_size = get_model_options().get('size')
+	model_name = state_manager.get_item('lip_syncer_model')
 	temp_audio_frame = prepare_audio_frame(temp_audio_frame)
 	crop_vision_frame, affine_matrix = warp_face_by_face_landmark_5(temp_vision_frame, target_face.landmark_set.get('5/68'), 'ffhq_512', (512, 512))
-	face_landmark_68 = cv2.transform(target_face.landmark_set.get('68').reshape(1, -1, 2), affine_matrix).reshape(-1, 2)
-	bounding_box = create_bounding_box(face_landmark_68)
-	bounding_box[1] -= numpy.abs(bounding_box[3] - bounding_box[1]) * 0.125
-	area_mask = create_area_mask(face_landmark_68, [ 'lower-face' ])
+	box_mask = create_static_box_mask(crop_vision_frame.shape[:2][::-1], state_manager.get_item('face_mask_blur'), state_manager.get_item('face_mask_padding'))
 	crop_masks =\
 	[
-		area_mask
+		box_mask
 	]
 
 	if 'occlusion' in state_manager.get_item('face_mask_types'):
 		occlusion_mask = create_occlusion_mask(crop_vision_frame)
 		crop_masks.append(occlusion_mask)
 
-	close_vision_frame, close_matrix = warp_face_by_bounding_box(crop_vision_frame, bounding_box, model_size)
-	close_vision_frame = prepare_crop_frame(close_vision_frame)
-	close_vision_frame = forward(temp_audio_frame, close_vision_frame)
-	close_vision_frame = normalize_close_frame(close_vision_frame)
-	crop_vision_frame = cv2.warpAffine(close_vision_frame, cv2.invertAffineTransform(close_matrix), (512, 512), borderMode = cv2.BORDER_REPLICATE)
+	if model_name.startswith('wav2lip'):
+		face_landmark_68 = cv2.transform(target_face.landmark_set.get('68').reshape(1, -1, 2), affine_matrix).reshape(-1, 2)
+		area_mask = create_area_mask(face_landmark_68, [ 'lower-face' ])
+		crop_masks.append(area_mask)
+		bounding_box = create_bounding_box(face_landmark_68)
+		bounding_box = prepare_bounding_box(bounding_box)
+		crop_vision_frame = process_wav2lip(crop_vision_frame, temp_audio_frame, bounding_box)
+	elif model_name == 'edtalk_256':
```

Comment:
edtalk comes first and we are not using elif

just `if model_name == 'edtalk_256':`